### PR TITLE
routing

### PR DESCRIPTION
**Pull request by LucasZibell**
## Summary

Navigation between two pages

## Screenshot
hello

### CODE REVIEW

File: trainingRN/src/constants/routes.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const HOME = 'Home';
+export const BOOK_LIST = 'BookList'
```
Comment (mvbattan) [inline trainingRN/src/constants/routes.js]: I'd export this as a whole, single object, so we can use
```
import routes from './constants/routes`
...
routes.HOME
```
I think this way we denote that these constants are indeed routes. What do you think ?

File: trainingRN/src/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  {
+    [HOME]: Todo,
+    [BOOK_LIST]: BookList
+  },
+  {
+    initialRouteName: HOME
+  }
+);
+
+export default class App extends React.Component {
```
Comment (mvbattan) [inline trainingRN/src/index.js]: I'd put `export default` at the bottom of this file

File: trainingRN/src/screens/bookList/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import { Text, View } from 'react-native';
+
+class BookList extends Component {
+  static navigationOptions = {
+    title: 'Book List',
```
Comment (mvbattan) [inline trainingRN/src/screens/bookList/index.js]: I'd extract this one to strings.js

File: trainingRN/src/screens/todo/index.js [hunk 4]
```diff
@@ -22,6 +23,10 @@ const todoList = [
 ]
 
 class TodoContainer extends Component {
+  static navigationOptions = {
+    headerTitle: 'Home'
```
Comment (mvbattan) [inline trainingRN/src/screens/todo/index.js]: I'd extract this to `strings.js`

**Review by mvbattan — COMMENTED** [review]
Looks great !!

File: trainingRN/src/index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createStackNavigator } from 'react-navigation';
+import Todo from './screens/todo';
+import BookList from './screens/bookList';
+import routes from './constants/routes';
+
+const RootStack = createStackNavigator(
+  {
+    [routes.HOME]: Todo,
+    [routes.BOOK_LIST]: BookList
```
Comment (mvbattan) [inline trainingRN/src/index.js:10]: Use a TabNavigator for these two

File: trainingRN/src/screens/bookList/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+import { BOOK_LIST} from '../../constants/constant';
+
+class BookList extends Component {
+  static navigationOptions = {
+    title: BOOK_LIST,
+  };
+  render() {
+    return (
+    <View />
```
Comment (mvbattan) [inline trainingRN/src/screens/bookList/index.js]: Check indentation

File: trainingRN/src/screens/bookList/index.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import { Text, View } from 'react-native';
+
+import { BOOK_LIST} from '../../constants/constant';
```
Comment (mvbattan) [inline trainingRN/src/screens/bookList/index.js]: Check spaces

File: trainingRN/src/constants/constant.js [hunk 8]
```diff
@@ -1,4 +1,7 @@
 export const DELETE_ALL = 'Delete all';
 export const ADD = 'Add';
 export const ADD_NEW_TASK = 'Add new task';
-export const X = 'X';
\ No newline at end of file
+export const X = 'X';
+export const GO_TO_DETAILS = 'Go to Details';
+export const BOOK_LIST = 'Book List';
+export const HOME = 'Home';
```
Comment (mvbattan) [inline trainingRN/src/constants/constant.js]: These should be at `constants/strings.js`

File: trainingRN/src/index.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  {
+    initialRouteName: routes.HOME
+  }
+);
+
+class App extends React.Component {
```
Comment (mvbattan) [inline trainingRN/src/index.js]: Use `Component`:
```
class App extends Component
```

File: trainingRN/src/index.js [hunk 10]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createStackNavigator } from 'react-navigation';
+import Todo from './screens/todo';
+import BookList from './screens/bookList';
```
Comment (mvbattan) [inline trainingRN/src/index.js]: Remember that folders that belong to React `Components` should be written with capital letters:
```
./screens/BookList
./screens/Todo
```

File: trainingRN/src/index.js (hunk 5 above)
Comment (LucasZibell) [reply trainingRN/src/index.js:10]: I'm using `stackNavigator` do you have an example on how to use TabNavigator?

Comment (mvbattan) [reply trainingRN/src/index.js:10]: [Docs](https://reactnavigation.org/docs/en/tab-based-navigation.html) may help here 😊

Comment (LucasZibell) [reply trainingRN/src/index.js:10]: Okey, since this is the only comment here i will apply this change in the other PR with the book list

**Review by mvbattan — APPROVED** [review]

**Pull request merged**